lua/nvim-treesitter/parsers.lua

```diff
@@ -47,6 +47,7 @@ for ft, lang in pairs {
   xsd = "xml",
   xslt = "xml",
   sbt = "scala",
+  neomuttrc = "muttrc",
```

Comment:
```suggestion
```

Reply:
@flatcap (the author of neomuttrc.vim) what is the difference between muttrc and neomuttrc? IMO, the latter should have more customized options?

Reply:
Oh, it is in Vim. Nevermind.